packages/extension-ui/src/Popup/Create.tsx

```diff
@@ -8,9 +8,7 @@ import { ActionContext, Address, Button, Header, Loading, TextArea } from '../co
 import { createAccount, createSeed } from '../messaging';
 import { Back, Name, Password } from '../partials';
 
-type Props = {};
-
-export default function Create (): React.ReactElement<Props> {
+export default function Create (): React.ReactElement<{}> {
```

Comment:
There . is actually some method behind this madness. When you decide to finally add props, the following can happen -

```js
interface Props {
  className?: string;
}

export default function Create ({ className }: Props): React.ReactElement<{}> {
```

Note how the actual return is still `{}`. The above passes without errors, warnings or linting, but is incorrect. Hence being explicit an always defining `Props` types, even when empty. It is some small protection against future rushing where things could very much get out of whack.

Reply:
got it, sounds good!

I actually changed this before merging from master, I was getting the `empty interface` linting error; I just saw you changed `Props` it to `type` :+1: